scripts/gitlab/build-unix.sh

```diff
@@ -26,7 +26,11 @@ mkdir -p artifacts
 cd artifacts
 mkdir -p $CARGO_TARGET
 cd $CARGO_TARGET
-cp ../../target/$CARGO_TARGET/release/{parity,parity-evm,ethstore,ethkey,whisper} .
+cp ../../target/$CARGO_TARGET/release/parity ./parity
+cp ../../target/$CARGO_TARGET/release/parity-evm ./parity-evm
+cp ../../target/$CARGO_TARGET/release/ethstore ./ethstore
+cp ../../target/$CARGO_TARGET/release/ethkey ./ethkey
+cp ../../target/$CARGO_TARGET/release/whisper ./whisper
```

Comment:
This should fix random paths on ARM build (I guess there is some crazy version of `cp` out there)